tests/tasks/el_repo_setup.yml

```diff
@@ -1,26 +0,0 @@
-# SPDX-License-Identifier: BSD-3-Clause
-- name: Fix CentOS6 Base repo
-  copy:
-    dest: /etc/yum.repos.d/CentOS-Base.repo
-    content: |
-      [base]
-      name=CentOS-$releasever - Base
-      baseurl=https://vault.centos.org/6.10/os/$basearch/
-      gpgcheck=1
-      gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
-
-      [updates]
-      name=CentOS-$releasever - Updates
-      baseurl=https://vault.centos.org/6.10/updates/$basearch/
-      gpgcheck=1
-      gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
-
-      [extras]
-      name=CentOS-$releasever - Extras
-      baseurl=https://vault.centos.org/6.10/extras/$basearch/
-      gpgcheck=1
-      gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
-  when:
-    - ansible_distribution == 'CentOS'
-    - ansible_distribution_major_version == '6'
-- include_tasks: enable_epel.yml
```

Comment:
Is this not needed for running the tests locally using a centos6 image? Thanks!

Reply:
> Is this not needed for running the tests locally using a centos6 image? Thanks!

It is needed for running the tests locally using a centos6 image.  But I don't think we want it in the test, in this form, since the CI environment already does this.

Reply:
@richm 
This is hiding information in a black box CI and I have to explain when anyone hit this problem locally.

Is there any problem caused by old code?